rotkehlchen/externalapis/monerium.py

```diff
@@ -221,15 +221,13 @@ def get_and_process_orders(self, tx_hash: EVMTxHash | None = None) -> None:
             if self.is_monerium_event_edited(event_notes=(event := events[0]).notes):
                 continue  # skip if the event is already edited
 
-            querystr = 'UPDATE history_events SET notes=? '
```

Comment:
why was this entire ode part removed?

Reply:
i replaced it with `edit_history_event()`. they're equivalent and it makes updating the cache easier.